exatomic/gaussian/output.py

```diff
@@ -835,7 +847,7 @@ def parse_frequency_ext(self):
         if not found:
             return
         else:
-            found = self.find(_renmode, _redisp, _refinfo, _remass, keys_only=True)
+            found = self.find(_renmode, _refinfo, keys_only=True)
```

Comment:
Is this a backwards incompatible change or was this previously unused?

Reply:
They are unused. The `_remass` is for the isotopic masses used. The `_redisp` is for the normal modes which is not what we are looking at in this method.